### ISSUE TITLE
The crawler reports "NOT_MODIFIED" even though dataset was never successfully crawled

### ISSUE THREAD

**Issue by kbraak** [gbif/crawler#8]
The crawler reports "NOT_MODIFIED" even though dataset was never successfully crawled. This should never happen.

In other words, it has no "NORMAL" crawl response ever recorded as can be seen here:

https://registry.gbif.org/web/#/dataset/1c714c57-da3e-457e-9752-edd1cd5e0463/crawl

**Comment by timrobertson100** [comment]
Indeed.  NOT_MODIFIED here is an HTTP HEAD call to see if the archive on the URL is newer than the one on the file system.  

In the meantime I have moved the DwC-A to 1c714c57-da3e-457e-9752-edd1cd5e0463.dwcaOLD so that it will force it to redownload the DwC-A and it will be indexed.

**Comment by timrobertson100** [comment]
It fails to index, with "The core data file contains non unique core IDs"
https://tools.gbif.org/dwca-reports/303-610796845517599109.html

**Comment by MattBlissett** [comment]
NOT_MODIFIED should probably become ABORT_NOT_MODIFIED where the previous crawl result was an ABORT.
